index.html

```diff
@@ -14,8 +14,98 @@
 				<li><a href="#">Contact</a></li>
 			</ul>
 		</nav>
-		
-
 	</div>
+	<section class="F-section"><p class="T-name">Amin Dream Team</p>
+		<img src="https://img.icons8.com/metro/208/000000/avengers.png">
+	</section>
+	<section class="sec2">
+		<div class="team">
+			<p class="t-mem" align="center">Team Members</p>
```

Comment:
again, go over all the classnames and change them to something more readable for other people.